libcudacxx/include/cuda/__nvtx/nvtx.h

```diff
@@ -32,6 +32,10 @@
 #if _CCCL_HAS_INCLUDE(<nvtx3/nvToolsExt.h>) && !defined(CCCL_DISABLE_NVTX) && !defined(NVTX_DISABLE) \
                       && (!_CCCL_COMPILER(NVHPC))                                                    \
                       && !_CCCL_COMPILER(NVRTC)
+
+// Since NVTX 3.2, the NVTX headers can declare themselves as system headers by declaring the following macro:
+#  define NVTX_AS_SYSTEM_HEADER
+
```

Comment:
I am a bit torn, whether this is something we can do.

If a user has already included nvtx3 this wont work. On the other hand I dont think it can really hurt so